.golangci.yaml

```diff
@@ -0,0 +1,85 @@
+issues:
+  exclude-rules:
+    - linters:
+        - staticcheck
+        - errcheck
+        - unconvert
+        - unparam
+        - dupword
+        - gosimple
+        - nilerr
+        - nilnil
+
+        # TODO(mael): add the following linters one by one and fix the issues
```

Comment:
Normally, I add a section at the top to ignore all linters that have errors (eg. https://github.com/cert-manager/cert-manager/blob/master/.golangci.yaml#L1-L26).